datajunction-server/datajunction_server/api/helpers.py

```diff
@@ -498,11 +497,6 @@ async def validate_cube(  # pylint: disable=too-many-locals
             message=("Metrics and dimensions must be part of a common catalog"),
         )
 
-    await validate_shared_dimensions(
```

Comment:
This validation happens later anyway, so skipping it here. This makes the sql build process faster and avoids the issue of dimension attributes that come from second-order reference links being skipped.